rest/blip_client_test.go

```diff
@@ -27,10 +27,12 @@ import (
 )
 
 type BlipTesterClientOpts struct {
-	ClientDeltas          bool // Support deltas on the client side
-	Username              string
-	Channels              []string
-	SendRevocations       bool
+	ClientDeltas           bool // Support deltas on the client side
+	Username               string
+	Channels               []string
+	SendRevocations        bool
+	SupportedBLIPProtocols []string
```

Comment:
We already have 'protocol' in place to set the client protocol, should either use that or replace that property with the slice if there's a reason for the test client to specify multiple supported protocols.